httpx/_client.py

```diff
@@ -731,6 +731,14 @@ def request(
 
         [0]: /advanced/#merging-of-configuration
         """
+        if cookies is not None:
+            message = (
+                "Setting per-request cookies=... is being deprecated, because "
```

Comment:
This would be better as
```suggestion
                "Setting per-request cookies=<...> is being deprecated, because "
```
for better language flow, I think (and similar to the `content=<...>` message in https://github.com/encode/httpx/pull/1573/files#diff-361c5daf6e5401b987414c619d003a20ea88f74fe9da50cbb2da72c337aeb290R152)